### PR TITLE
LB -> Ingress

### PR DESCRIPTION
**Pull request by posix4e**
The default configuration used a LB. It seems to have worked. I'm going to switch this to use the default ingress however.

### CODE REVIEW

File: config/deployment.yml [hunk 1]
```diff
@@ -30,16 +30,74 @@ spec:
...
 ---
-apiVersion: v1
-kind: Service
+apiVersion: networking.k8s.io/v1
+kind: Ingress
 metadata:
-  name: python-example-service
+  annotations:
+    cert-manager.io/issuer: letsencrypt-prod
+    kubectl.kubernetes.io/last-applied-configuration: |
```
Comment (ryanobjc) [inline config/deployment.yml:38]: do we need this here?

File: config/deployment.yml [hunk 2]
```diff
@@ -30,16 +30,74 @@ spec:
...
+      {"apiVersion":"networking.k8s.io/v1","kind":"Ingress","metadata":{"annotations":{"cert-manager.io/issuer":"letsencrypt-prod","nginx.ingress.kubernetes.io/auth-signin":"https://$host/oauth2/start?rd=$escaped_request_uri","nginx.ingress.kubernetes.io/auth-url":"https://$host/oauth2/auth"},"name":"external-auth-oauth2","namespace":"default"},"spec":{"ingressClassName":"nginx","rules":[{"host":"test`.cursivestrategies.com","http":{"paths":[{"backend":{"service":{"name":"demo","port":{"number":80}}},"path":"/","pathType":"Prefix"}]}}],"tls":[{"hosts":["test.cursivestrategies.com"],"secretName":"oauth2-proxy-tls"}]}}
+    nginx.ingress.kubernetes.io/auth-signin: https://$host/oauth2/start?rd=$escaped_request_uri
+    nginx.ingress.kubernetes.io/auth-url: https://$host/oauth2/auth
+  creationTimestamp: "2022-02-11T21:03:04Z"
```
Comment (ryanobjc) [inline config/deployment.yml:42]: ditto basically any internally driven k8 fields too

**Review by ryanobjc — COMMENTED** [review]
Is your intent on replicating the current ingress that I'm using for the jupyter lab setup?  I control the cursivestrategies.com on gandi dns so we can do that too.

File: config/deployment.yml (hunk 1 above)
Comment (posix4e) [reply config/deployment.yml:38]: No idea i copied and pasted what i have

File: config/deployment.yml (hunk 2 above)
Comment (posix4e) [reply config/deployment.yml:42]: no idea. Probably not needed. Want me to remove?

**Pull request merged**